src/lib/storage/fitted_attribute_vector.cpp

```diff
@@ -0,0 +1,36 @@
+#include <limits>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "fitted_attribute_vector.hpp"
+
+namespace opossum {
+
+template <typename T>
+ValueID FittedAttributeVector<T>::get(const size_t i) const {
+  return ValueID(_attribute_vector.at(i));
+}
+
+template <typename T>
+void FittedAttributeVector<T>::set(const size_t i, const ValueID value_id) {
+  _attribute_vector.insert(_attribute_vector.cbegin() + i, value_id);
```

Comment:
As posted in the Piazza, the desired behaviour here is a "set" (with replace) and not "insert".